includes/Hooks.php

```diff
@@ -73,149 +78,120 @@ public static function registerExtension() {
 	/**
 	 * Handler for PageContentSaveComplete hook
 	 *
-	 * @param WikiPage $wikiPage   WikiPage modified
-	 * @param User     $agent      User performing the modification
-	 * @param Content  $content    New content, as a Content object
-	 * @param string   $summary    Edit summary/comment
-	 * @param boolean  $isMinor    Whether or not the edit was marked as minor
-	 * @param boolean  $isWatch    (No longer used)
-	 * @param string   $section    (No longer used)
-	 * @param integer  $flags      Flags passed to WikiPage::doEditContent()
-	 * @param Revision $revision   Revision object of the saved content.  If the save did not result in the creation
-	 *                             of a new revision (e.g. the submission was equal to the latest revision), this
-	 *                             parameter may be null (null edits, or "no-op").
-	 * @param Status   $status     Status object about to be returned by doEditContent()
-	 * @param integer  $baseRevId  the rev ID (or false) this edit was based on
-	 * @param integer  $undidRevId the rev ID (or 0) this edit undid - added in MW 1.30
-	 *
+	 * @param WikiPage $wikiPage WikiPage modified
+	 * @param UserIdentity $user
+	 * @param string $summary Edit summary/comment
+	 * @param int $flags Flags passed to WikiPage::doEditContent()
+	 * @param RevisionRecord $revisionRecord
+	 * @param EditResult $editResult
+	 * @return bool True
+	 * @throws Identifier\InvalidIdentifierException
+	 * @throws LoaderError
+	 * @throws MWException
+	 * @throws RuntimeError
+	 * @throws SyntaxError
 	 * @see http://www.mediawiki.org/wiki/Manual:Hooks/PageContentSaveComplete
-	 *
-	 * @return boolean True
 	 */
 	public static function onPageContentSaveComplete(
```

Comment:
https://www.mediawiki.org/wiki/Manual:Hooks/PageContentSaveComplete - removed in 1.37